chart/sysdig-agent/templates/priorityclass.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: "{{ .Values.metadata.name }}-ds-priority"
+  namespace: {{ .Release.Namespace }}
+value: 1000000
```

Comment:
Do you think the priority should be lower than logging so any errors are logged? Maybe it doesn't matter?

Reply:
This is a good question and I was not sure myself about the correct priority. I'm fine with lowering it a bit

Reply:
@shemau any thoughts here?

Reply:
@kulmannf I think set it to `999999` so it's right after logs priority-wise. It probably doesn't make too much of a difference anyway

Reply:
@daniel-butler-irl Updated the value to 999999